x-pack/test/api_integration/apis/aiops/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ loadTestFile }: FtrProviderContext) {
+  describe('AIOps', function () {
+    this.tags(['mlqa', 'walterra']);
```

Comment:
Looks like you left in `walterra` by mistake. Also I think we should now be using `ml` and mlqa`.

Reply:
Fixed in [f5b98ee](https://github.com/elastic/kibana/pull/131317/commits/f5b98eed2e4dbe096e1bb0f7bc912be8ee2d5dd2).